### DISCUSSION TITLE
Use router. push() page without refreshing

### DISCUSSION THREAD

**Discussion by linuxzcy** [nuxt/nuxt#24331] [Questions]
### Environment

dev or test

### Reproduction

bug

### Describe the bug

Use router. push() page without refreshing ？Nuxt routing redirects to other pages, how to refresh page routing without initializing pinia data

**Comment by danielroe** [comment]
Have you tried `useRouter().push({ path: '/my/path', force: true })`?

**Reply by linuxzcy** [reply]
I haven't tried it before, I haven't seen the force parameter in router. push()? I didn't notice
